examples/clone.py

```diff
@@ -27,7 +28,10 @@
 elif len(args) < 3:
     target_path = basename(args[1].split(":")[-1])
     if target_path[-4:] == ".git":
-        target_path = target_path[:-4]
```

Comment:
This causes porcelain.clone to be invoked twice.

Reply:
Yes, I know, I have a correction.

I put this feature in the new Pull requests.

#828